### DISCUSSION TITLE
Toggling settings, Open Shell vs Win+X

### DISCUSSION THREAD

**Discussion by TJAugustus** [Open-Shell/Open-Shell-Menu#627] [Q&A]
So I was in Open-Shell's settings a few minutes ago, and checked the box pointed out below.  Now I can't find a way to get back in to uncheck it, because I disabled the only way I know how to get into Open-Shell's settings.

I tried reinstalling Open-Shell, but the settings are saved somewhere in the registry that I can't find, so reinstalling doesn't change anything.  I'd like a simpler solution anyway.  How do I get into Open-Shell's settings, other than right-clicking start?

Thanks!
![56336743-0eac0580-61aa-11e9-95fa-f4a9a0688c61](https://user-images.githubusercontent.com/78280609/106369239-86f2a500-6304-11eb-9dab-84f2d12637a8.JPG)

**Comment by bonzibudd** [comment]
C:\Program Files\Open-Shell\Start Menu Settings.lnk. There should also be a shortcut in the Start menu folder, unless you removed it during installation.

**Reply by TJAugustus** [reply]
Got it, thanks!
